jwt_rsa/token.py

```diff
@@ -13,40 +16,51 @@
 else:
     DateType = Union[timedelta, datetime, float, int, type(Ellipsis)]
 
+
 R = TypeVar("R")
+DAY = 86400
 
 
 class JWT:
     __slots__ = (
         "__private_key", "__public_key", "__jwt",
         "__expires", "__nbf_delta", "__algorithm",
+        "__algorithms",
     )
 
-    DEFAULT_EXPIRATION = 86400 * 30  # one month
+    DEFAULT_EXPIRATION = 31 * DAY  # one month
```

Comment:
`timedelta(days=31).total_seconds()`

Reply:
It was little bit refactored not rewritten